app/matchers/bulkrax/application_matcher_decorator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+# OVERRIDE Bulkrax 4.4.2 Application Parser here to make sure atla custom parse methods get added
+Bulkrax::ApplicationMatcher.class_eval do
```

Comment:
as info for the reviewer, using modules with the `prepend` syntax didn't work for some reason. 

re: [this thread](https://assaydepot.slack.com/archives/G0313NK5NMA/p1682544588165129).